internal/serve/httphandler/login_handler.go

```diff
@@ -68,11 +69,28 @@ func (h LoginHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if !h.ReCAPTCHADisabled {
+	user, err := h.AuthManager.GetUserByEmail(ctx, reqBody.Email)
+	if err != nil {
+		if errors.Is(err, auth.ErrUserNotFound) {
+			// If we don't find the user by email, we just return an ok response
+			// to prevent malicious client from searching accounts in the system
+			log.Ctx(ctx).Errorf("Email in request not found: %s", reqBody.Email)
+		} else {
+			httperror.InternalError(ctx, "Getting user by email", err, nil).Render(rw)
```

Comment:
🚨  We shouldn't return any hints of what went wrong during the authentication, especially before recaptcha, at the risk of providing too much information to a bad actor that's trying to brute force the system.

```suggestion
			httperror.InternalError(ctx, "", err, nil).Render(rw)
```